### PR TITLE
USHIFT-2053: only install osbuild from COPR on RHEL 9.3

### PR DESCRIPTION
**Pull request by dhellmann**
The version on RHEL 9.2 works OK for building only 9.2 images, so
allow it to be used.

/assign @ggiguash @pmtk

### CODE REVIEW

File: scripts/image-builder/configure.sh [hunk 1]
```diff
@@ -13,20 +13,31 @@ auth_file_path = "/etc/osbuild-worker/pull-secret.json"
...
+    sudo dnf copr enable -y @osbuild/osbuild-composer rhel-9-"$(uname -m)"
+    composer_packages="
+    osbuild-composer-96-1.20231213092324044071.main.10.g991293a89.el9
+    osbuild-101-1.20231213161521815551.main.24.g05c0fd31.el9
+    "
+else
+    # osbuild from the standard RPM repository, limits about which os
+    # versions can be built apply
+    composer_packages="osbuild-composer osbuild"
+fi
```
Comment (ggiguash) [inline scripts/image-builder/configure.sh:32]: Why is this necessary? We're supposed to be using RHEL 9.3 for devenv now.

Comment (pmtk) [reply scripts/image-builder/configure.sh:32]: Yes, we should makes sure that all 4.14, 4.15, and main branches use rhel93 as hypervisor
